portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
+html, body {
+  background-color: #F5F5F5;
+  font-family: helvetica, sans-serif;
+}
+
+#index{
+    position: absolute;
+    margin: 0px;
+    overflow: hidden;
+    top: 0px;
+    bottom: 0px;
+    right: 0px;
+    left: 0px;
+    border-style: solid;
+    border-width: 15px;
+    border-color: black;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  height: 100%;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
 }
+
+.container{
+    display: inline;
+}
+.titleImage {
+    max-height: 500px;
+    max-width: 500px;
+    object-fit: cover;
+    right: 0px;
+    bottom: 0px;
+    position: absolute;
+}
+
+/* Title Font*/
+h1{
+    color:black;
```

Comment:
extra space before \*\/
extra space before \{
extra spece after \: (also applies to below lines)
Indent by 2 spaces at a time: https://google.github.io/styleguide/htmlcssguide.html#Indentation